client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -167,3 +131,56 @@
 .sticky-inner-wrapper {
   z-index: 1;
 }
+
+.question_type {
+  &--select {
+    border-color: color('white', 10);
+    background-color: color('teal', 40);
+    color: color('white', 10);
+    background-image: url(triangle(color('white', 10), $size: 1.5));
+    padding-right: 1.7rem;
+
+    &:focus {
+      outline-color: color('white', 10);
+    }
+  }
+}
+
+.feature_select {
+  &--list {
+    display: flex;
+    margin-bottom: get('rhythm');
+    border-bottom: 1px solid color('white', 10);
+  }
+
+  &--list_item {
+    margin-right: 1rem;
+
+    &::before {
+      content: '';
+      padding: 0;
+    }
+  }
+
+  &--list_button {
+    border: none;
+    border-radius: 0;
+    border-color: color('white', 10);
+    color: color('teal', 10);
+    font-weight: get('weight light');
+    padding: 0 0 0.2rem 0;
+    margin-right: get('rhythm');
+    line-height: 2.5rem;
+    transition: none;
+
+    &--active {
+      border-bottom: 0.2rem solid color('white', 10);
+      color: color('white', 10);
+    }
+
+    &:active, &:focus, &:hover {
+      color: color('white', 10);
+      border-bottom-color: currentColor;
+    }
+  }
+}
```

Comment:
I'm really happy to see this BEM scss pattern. Reads great.

Reply:
👍